Library/Homebrew/cmd/test-bot.rb

```diff
@@ -445,6 +445,10 @@ def formula formula_name
         test "brew", "test", "--verbose", formula_name if formula.test_defined?
         testable_dependents.each do |dependent|
           unless dependent.installed?
+            conflicts = dependent.conlicts.map { |c| Formulary.factory(c.name) }.select { |f| f.installed? }
```

Comment:
I haven't yet test it as I don't know which command should be invoked in `test-bot`. I tried `brew test-bot --ci-pr --skip-setup <pr-url>`. But it didn't the real test at all.